.github/workflows/update_dependencies.yml

```
@@ -1,38 +1,8 @@
-name: Update dependencies
+name: Update Python Dependencies
 
 on:
     workflow_dispatch:
 
 jobs:
-    update_dependencies:
-        runs-on: ubuntu-latest
-        steps:
-            - name: "Checkout"
-              uses: actions/checkout@v3
-              with:
-                  ref: main
-                  fetch-depth: 0
-            - name: Set up Python 3.8
-              id: py
-              uses: actions/setup-python@v2
-              with:
-                  python-version: 3.8
-                  cache: pip
-                  cache-dependency-path: "requirements/*.txt"
-            - name: Update dependencies
-              run: |
-                  pip install pip-tools
-                  make update-deps
-            - name: Commit changes and open PR
-              run: |
-                  export WEEK="W$(date +%V)"
-                  git config user.name github-actions
-                  git config user.email github-actions@github.com
-                  git switch -c "dependency-updates-$WEEK"
-                  git add requirements/*.txt 
-                  git commit -m "$WEEK dependency updates (automated)"
-                  git push origin "dependency-updates-$WEEK"
-            - env:
-                  GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
-              run: |
-                  gh pr create --fill --label 'auto-pr,dependencies'
+    update_python_dependencies:
+        uses: SFDO-Tooling/.github/.github/workflows/update_python_dependencies.yml
```

±: Pass in the python version (see SFDO-Tooling/.github#4 review [comment](https://github.com/SFDO-Tooling/.github/pull/4#discussion_r883673664)):
```suggestion
        uses: SFDO-Tooling/.github/.github/workflows/update_python_dependencies.yml
        with:
                python-version: 3.8
```